pkgs/development/compilers/dotnet/build-dotnet.nix

```diff
@@ -47,6 +48,10 @@ in stdenv.mkDerivation rec {
     lttng-ust_2_12
   ]);
 
+  # Fixes cross
+  nativeBuildInputs = [ autoPatchelfHook ];
```

Comment:
Why do we need to autopatchelf when compiling from source?

Reply:
I'm not entirely sure why, but without it a cross compiled binary fails as follows:
```console
~/nix/nixpkgs > nix build -Lf . pkgsCross.aarch64-multiplatform.dotnet-sdk_6
~/nix/nixpkgs > ./result/bin/dotnet
./result/bin/dotnet: error while loading shared libraries: libstdc++.so.6: cannot open shared object file: No such file or directory
```

Reply:
hmmm. I am not sure we should run patchelf on .NET binaries. That usually causes trouble.

Reply:
I haven't experienced any issues with it so far, do you know of any specific problems?